### PR TITLE
Portfolio: application deploy

### PR DESCRIPTION
**Pull request by basem909**
in this part of the portfolio project the project was deployed on the internet using Github pages 
the link was to the page was added to the README.md file you will find it firstly below the links section within the file 
the page has passed the deployment and linters checks and working fine online

### CODE REVIEW

**Comment by basem909** [comment]
add the link on GitHub pages to the readme.md file

File: README.md [hunk 1]
```diff
@@ -20,12 +20,13 @@ The forth part is applying media queries to the site to make the mobile version
 
 - https://basem909.github.io/microverse-portofolio-project/
```
Comment (Whoistolu) [inline README.md]: - [ ] [REQUIRED] Kindly ensure that the website is deployed successfully using GitHub pages.
Currently, this is the error message we have on the page:
Screenshot from 2022-03-28 22-06-49

**Review by Whoistolu — COMMENTED** [review]
- [ ]  _Check the inline comments under the review for sections to improve._

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @basem909 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] Kindly add a description to your PR in order to make your work look more professional.
- [ ] [REQUIRED] Kindly update the README, add the MIT License please as it grants permission to use the code and modify it.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (basem909) [reply README.md]: Hi Whoistolu,
It is already deployed successfully and working here you could give it a look in the actions section it's all right

**Comment by basem909** [comment]
hi @Whoistolu 
would you please tell me how to edit the pull request description without opening a new one because I found out that I wrote it in the title instead of the description?!
would you please tell me what the MIT license is ?

**Comment by Whoistolu** [comment]
> hi @Whoistolu would you please tell me how to edit the pull request description without opening a new one because I found out that I wrote it in the title instead of the description?! would you please tell me what the MIT license is ?

Hello @basem909 
Screenshot from 2022-03-28 22-42-48
Here is where the description is supposed to be. Right beside the smiley icon is the options icon, click on it to edit and add a description.

And for the MIT License, is a free license software. The MIT License grants the software end-user rights such as copying, modifying, merging, distributing code. Check the [Microverse README Template](https://github.com/microverseinc/readme-template) for the link.

**Comment by basem909** [comment]
this is a screenshot of the page working fine and you could see the link on the URL field
Screen Shot 2022-03-28 at 7 04 54 PM

File: README.md (hunk 1 above)
Comment (AmaduKamara) [reply README.md]: - [ ] Kindly update your website live demo link in your README.md file
- [ ] I noticed you made a little mistake on the link you used in your README.md file.
- You used this `https://basem909.github.io/microverse-portofolio-project/` and in the photo, you used to reply to the previous reviewer, there is a dash after project in your URL. 
- https://basem909.github.io/microverse-portofolio-project-/
- [ ] Please update your README with this last link with the dash after `project `👍 
- Please see the screenshot below.
image

- [ ] I recommend you always use the correct link wherever you need to, test if all is working before pushing for a review.

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️

Hi @basem909, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Suggested changes
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi @basem909,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**